pep/app/utils/dom.ts

```diff
@@ -79,3 +79,18 @@ export function parseXML(input: string): XMLDocument | Error {
         return new Error('Unable to parse xml');
     }
 }
+
+/**
+ * Load the XSLT doc to do the parsing of the xml
+ *
+ * @return {XSLT}
+ */
+export function loadXSLT(this: any): Promise<Document | null> {
+    return new Promise((resolve) => {
+        fetch('/xmlToHtml.xslt').then(async (response) => {
+            const text = await response.text();
+            const xml = parseXML(text);
+            resolve(xml as Document);
+        });
+    });
+}
```

Comment:
I think you'll want to do `import fetch from 'fetch';` in this file so it uses the version of `fetch()` coming from ember-fetch which is isomorphic and works in node as well.

also, don't have to change this if you don't want to, but I think this whole function could be async/await based couldn't it? e.g.

```ts
export async function loadXSLT(this: any): Promise<Document | null> {
    const response = await fetch('/xmlToHtml.xslt');
    const text = await response.text();
    const xml = parseXML(text);
    return xml as Document;
}
```